packages/attack-http-requests/src/index.test.ts

```diff
@@ -0,0 +1,43 @@
+import HttpAttack from './index';
+import got from 'got';
+
+const URL = 'http://postman-echo.com';
+
+const makeRequest = async () => {
+  try {
+    await got.get(`${URL}/get?foo1=bar1&foo2=bar2`);
+    return true;
+  } catch (e) {
+    return false;
+  }
+};
+
+const createXRequests = (num: number) =>
+  Promise.all(new Array(num).fill(makeRequest()));
+
+test('If configuration disabled, requests always return true', async () => {
```

Comment:
Could we follow a consistent test strategy? The other modules are like this:

```ts
describe('when configuration is disabled' () => {
  it('always returns true', async () => {
    // ...
  });
});
```

This is far more extensible for future tests and allows beforeEach and afterEach groupings

Reply:
Updated.